sdk/communication/Azure.Communication.Administration/src/Models/InternalPhoneNumberReservationOperation.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Communication.Administration.Models
+{
+    internal class InternalPhoneNumberReservationOperation
+    {
```

Comment:
This type is not only for reservation right? Then maybe we should call it `PhoneNumberLongRunningOperation`